src/System.Threading.Channels/src/System.Threading.Channels.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <ProjectGuid>{1DD0FF15-6234-4BD6-850A-317F05479554}</ProjectGuid>
+    <RootNamespace>System.Threading.Channels</RootNamespace>
+    <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
```

Comment:
You should remove this given that you don't have a intellisense documentation file.

Reply:
> You should remove this given that you don't have a intellisense documentation file.

@weshaggard, I don't understand: this line is what causes it to have an IntelliSense doc file.  Without this line, the compiler won't validate the XML comments and produce such a file.

Reply:
For some reason I was thinking of this as input and not output. If we have xml doc comments in the code then this is fine.